### DISCUSSION TITLE
Run-time error when running in debug mode

### DISCUSSION THREAD

**Discussion by SuperComeng** [microsoft/vcpkg#19294] [Q&A]
Hi, everyone :)

I have installed libraries for linear algebra, 3D viewer, optimization, and so on, in order to make a visual SLAM project.
When I run a sample source in Visual Studio 2019 x64 debug mode on Windows10 , some libraries yield run-time error.
For example, in case of armadillo,
arma::mat A;
ifstream fin("any_matrix_file.txt");
A.load(fin);  <--- in debug mode: run-time error, in release mode: working well
In case of pangolin,
Run-time error occurs when I click on a pangolin OpenGL window to move or rotate a cube object...

However, in release mode, my program is working well. There are no run-time error.

Does anyone know why there is run-time error in debug mode?

**Comment by mathisloge** [comment]
So the `any_matrix_file.txt` have to be in the root dir of your working dir. (The dir from where the executable will be invoked). So it looks like the file `any_matrix_file.txt` is missing in the Debug folder. 

However, this doesn't look like a vcpkg issue.

**Reply by SuperComeng** [reply]
Thank you. I'll check my working dir path and file location. :)
